src/Sarif/Visitors/InsertOptionalDataVisitor.cs

```diff
@@ -214,6 +218,34 @@ public override Result VisitResult(Result node)
                 node.Guid = Guid.NewGuid().ToString(SarifConstants.GuidFormat);
             }
 
+            if (_dataToInsert.HasFlag(OptionallyEmittedData.GitBlameInformation))
+            {
+                // add git blame information to the result
+                string filePath = GetFilePath(node.Locations[0].PhysicalLocation.ArtifactLocation);
+
+                if (filePath != null)
+                {
+                    IEnumerable<SarifTransformerUtilities.IBlameHunk> blameHunks = SarifTransformerUtilities.ParseBlameInformation(
+                                                                                        _gitHelper.GetBlame(filePath));
+
+                    Region region = node.Locations[0].PhysicalLocation.Region;
+                    if (region != null)
+                    {
+                        foreach (SarifTransformerUtilities.IBlameHunk blameHunk in blameHunks)
+                        {
+                            if (!blameHunk.ContainsLine(region.StartLine))
+                            {
+                                continue;
+                            }
+                            node.SetProperty(nameof(CommitSha), blameHunk.CommitSha);
```

Comment:
>nameof(CommitSha) [](http://example.com/codeflow?start=45&length=17)

replace by CommitSha const (just like the others below)

Reply:
Fixed.